torch/_export/serde/serialize.py

```diff
@@ -1355,7 +1404,12 @@ def deserialize_graph(self, serialized_graph: Graph) -> torch.fx.Graph:
 
         return self.graph
 
-    def deserialize_node(self, serialized_node: Node, target: Callable) -> None:
+    def deserialize_node(
+        self,
+        serialized_node: Node,
+        target: Callable,
+        co_fileinfo_ordered_list: List[CoFileInfo],
```

Comment:
instead of passing co_fileinfo_ordered_list through the functions, maybe it's better to store it in GraphDeserializer so that you can get it from self.co_fileinfo_ordered_list

Reply:
this is how we construct the GraphModuleDesrializer: 
```
        res = (
            GraphModuleDeserializer()
            .deserialize(
                exported_program.graph_module,
                state_dict,
                constants,
                symbol_name_to_range,
            )
        )
```
do you suggest me to pass in exported_program.graph_module.graph (or co_fileinfo_ordered_list of the graph) as a param to the init of GraphModuleDeserializer?

Reply:
No, like somewhere in [`deserialize_graph`](https://fburl.com/code/hhbc1yac) you can store graph.co_fileinfo_ordered_list into self.co_fileinfo_ordered_list. That way you don't have to pass this variable everywhere else

Reply:
gotcha, yea we can do that, tho I feel this also creates some internal state that is less obvious? I can try it